packages/espressocash_app/lib/features/currency/models/amount.dart

```diff
@@ -51,6 +54,8 @@ sealed class Amount with _$Amount {
 
   bool get isZero => decimal == Decimal.zero;
 
+  bool get isUnknown => value < 0;
```

Comment:
No, negative amount can be totally valid value, it shouldn't have a special meaning. What's wrong with having `null` as "unknown amount"?

Reply:
This should be removed.

Reply:
> This should be removed.

sorry, I must have forgotten to add the file to the commit.